EpsilonDriverControls/Inc/DriverControls.h

```diff
@@ -50,6 +53,9 @@ extern ADC_HandleTypeDef hadc2;
 extern osMessageQId canQueue;
 extern osPoolId canPool;
 extern uint8_t auxBmsInputs[2];
+extern float   motorVelocityInput[2];
```

Comment:
I think 2 separate floats makes more sense, given these are for 2 different motors and are not part of the same input.